check_newest_file_age

```diff
@@ -29,11 +29,16 @@ The following arguments are accepted:
                   directory will be checked for the newest created file in that
                   directory.
 
+  -f              (Optional) Filename pattern with wildcard ? * and [0-9][a-z]
+                  It's used for select specific files.  Defaults is '*'.
+
   -w              (Optional) Generate a warning message if the last created
                   file is older than this value.  Defaults to 26 hours.
+                  To disable this treshold, set -w '-'
```

Comment:
The feature to suppress warning/critical messages is out of scope for this pull request, and should be removed. I'm not even sure it's a feature I would accept, but would consider it in another pull request.